components/match2/wikis/halo/match_group_input_custom.lua

```diff
@@ -312,19 +321,69 @@ function matchFunctions.getVodStuff(match)
 	if match.preview then links.preview = match.preview end
 	if match.esl then links.esl = 'https://play.eslgaming.com/match/' .. match.esl end
 	if match.faceit then links.faceit = 'https://www.faceit.com/en/halo_infinite/room/' .. match.faceit end
+	if match.halodatahive then links.halodatahive = 'https://halodatahive.com/Tournament/Detail/' .. match.halodatahive end
 	if match.stats then links.stats = match.stats end
 
 	return match
 end
 
 function matchFunctions.getExtraData(match)
+	local casters = {}
+	for key, name in Table.iter.pairsByPrefix(match, 'caster') do
+		table.insert(casters, CustomMatchGroupInput._getCasterInformation(
+			name,
+			match[key .. 'flag'],
+			match[key .. 'name']
+		))
+	end
+	table.sort(casters, function(c1, c2) return c1.displayName:lower() < c2.displayName:lower() end)
+
 	match.extradata = {
 		mvp = matchFunctions.getMVP(match),
-		isconverted = 0
+		casters = Table.isNotEmpty(casters) and Json.stringify(casters) or nil,
 	}
 	return match
 end
 
+function CustomMatchGroupInput._getCasterInformation(name, flag, displayName)
```

Comment:
if we are adding it to lots of wikis we might want to add it to commons base (together with lines331-340